platform/ios/src/MGLMapView.h

```diff
@@ -352,6 +352,15 @@ MGL_EXPORT IB_DESIGNABLE
  */
 @property (nonatomic, assign) MGLMapViewPreferredFramesPerSecond preferredFramesPerSecond;
 
+/**
+ A Boolean value indicating whether the map should prefetch tiles.
+ 
+ Loads tiles at a lower zoom-level to pre-render a low resolution preview while more detailed tiles are loaded.
+ 
+ Default is YES.
```

Comment:
The developer may be unfamiliar with the concept of prefetching simplified tiles, so here’s an attempt at making the mechanism easy to understand:

> When this property is set to `YES`, the map view prefetches loads tiles designed for a low zoom level and displays them until receiving more detailed tiles for the current zoom level. The prefetched tiles typically contain simplified versions of each shape, improving the map view’s perceived performance. The default value of this property is `YES`.

Reply:
Cool. That's a better one. I'll change to that.